genesis/src/main.rs

```diff
@@ -15,7 +15,7 @@ use std::error;
 //pub const BOOTSTRAP_LEADER_LAMPORTS: u64 = 3;
 // TODO: Until https://github.com/solana-labs/solana/issues/2355 is resolved the bootstrap leader
 // needs N lamports as its vote account gets re-created on every node restart, costing it lamports
-pub const BOOTSTRAP_LEADER_LAMPORTS: u64 = 1_000_000;
+pub const BOOTSTRAP_LEADER_LAMPORTS: u64 = 10;
```

Comment:
Can you update the comment in the 3 lines above to reflect the latest state of the codebase